docs/operations/clone_api.md

```diff
@@ -0,0 +1,157 @@
+# Clone API
+
+The `clone.kubevirt.io` API Group defines resources for cloning KubeVirt objects. Currently, the only supported cloning
+type is `VirtualMachine`, but more types are planned to be supported in the future (see future roadmap below).
+
+## Prerequesites
+
+### Snapshot / Restore
+
+Under the hood, the clone API relies upon Snapshot & Restore APIs. Therefore, in order to be able to use the clone API,
+please see [Snapshot & Restore prerequesites](./snapshot_restore_api.md/#prerequesites).
+
+### Snapshot Feature Gate
+
+Currently, clone API is gaurded by Snapshot feature gate. The
+[feature gates](./activating_feature_gates.md#how-to-activate-a-feature-gate)
+field in the KubeVirt CR must be expanded by adding the `Snapshot` to it.
+
+## The clone object
+
+Firstly, as written above, the clone API relies upon Snapshot & Restore APIs under the hood. Therefore, it might be helpful
+to look at [Snapshot & Restore](./snapshot_restore_api.md) user-guide page for more info.
+
+### VirtualMachineClone object overview
+
+In order to initiate clone, a `VirtualMachineClone` object (CRD) needs to be created on the cluster. An example
+for such an object is:
+```yaml
+kind: VirtualMachineClone
+apiVersion: "clone.kubevirt.io/v1alpha1"
+metadata:
+  name: testclone
+
+spec:
+  # source & target definitions
+  source:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-cirros
+  target:
+    apiGroup: kubevirt.io
+    kind: VirtualMachine
+    name: vm-clone-target
+
+  # labels & annotations definitions
+  labelFilters:
+    - "*"
+    - "!someKey/*"
+  annotationFilters:
+    - "anotherKey/*"
+
+  # other identity stripping specs:
+  newMacAddresses:
+    interfaceName: "00-11-22"
+  newSMBiosSerial: "new-serial"
+```
+
+In the next section I will go through the different settings to elaborate them.
+
+#### Source & Target
+
+The source and target indicate the source/target API group, kind and name. A few important notes:
+* Currently, the only supported kind is `VirtualMachine`, but more types are expected to be supported in the future.
+  See "future roadmap" below for more info.
+* The target name is **optional**. If unspecified, the clone controller will generate a name for the target automatically.
```

Comment:
In the same naamespace I guess?

Reply:
Yes, currently only same-namespace-clones are supported, I'll add that